### DISCUSSION TITLE
balenaEtcher on Ubuntu (STOP DO NOT USE!!)

### DISCUSSION THREAD

**Discussion by bretthl** [memtest86plus/memtest86plus#223] [General]
I noticed that you recommend balenaEtcher for Ubuntu.  I caution you that this application is not needed on Ubuntu and does not run properly.  I don't know who is behind this piece of software (spyware?).  I encourage you to change the ISO to USB method for Ubuntu to Gnome Multi Writer (sudo apt install gnome-multi-writer) since it is stable and in the Ubuntu repository.

![image](https://user-images.githubusercontent.com/11840200/209359794-6da4dd83-7716-417e-b3b9-68b1540384ac.png)

**Comment by x86fr** [comment]
Balena Etcher is a well-known, free open-source software recommanded by many linux distro to burn their ISO, including Ubuntu.

**Comment by JacobCarrell** [comment]
Or just use DD.
